src/labkey/Experiment.ts

```diff
@@ -597,4 +597,45 @@ export function saveRuns(options: SaveRunsOptions): XMLHttpRequest {
         success: getCallbackWrapper<Run[]>(getOnSuccess(options), options.scope, false, createRuns),
         failure: getCallbackWrapper(getOnFailure(options), options.scope, true)
     });
-}
\ No newline at end of file
+}
+
+export interface GenIdActionsOptions extends RequestCallbackOptions {
+    containerPath?: string
+    rowId: number
+    kindName: 'SampleSet' | "DataClass"
```

Comment:
Can we use `SampleType` instead of `SampleSet`?

Reply:
SampleTypeDomainKind.NAME is SampleType currently